tools/tests/tester/fixtures/evals/error/prologue_error.yaml

```diff
@@ -16,7 +16,6 @@ prologues:
     overall:
       result: ERROR
       message: no such index [does_not_exists]
-      error: Request failed with status code 404
```

Comment:
These aren't actually errors per-se, they get parsed and properly reflected in the failures of the test.